services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainConfiguration.java

```diff
@@ -71,28 +74,24 @@ private Builder() {}
     public PowchainConfiguration build() {
       validate();
       return new PowchainConfiguration(
-          eth1Endpoint.orElse(null),
-          depositContract,
-          depositContractDeployBlock,
-          eth1LogsMaxBlockRange);
+          eth1Endpoints, depositContract, depositContractDeployBlock, eth1LogsMaxBlockRange);
     }
 
     private void validate() {
-      if (eth1Endpoint.isPresent()) {
+      if (!eth1Endpoints.isEmpty()) {
         checkNotNull(
             depositContract,
             "Eth1 deposit contract address is required if an eth1 endpoint is specified.");
       }
     }
 
-    public Builder eth1Endpoint(final String eth1Endpoint) {
-      checkNotNull(eth1Endpoint);
-      return eth1Endpoint(Optional.of(eth1Endpoint));
-    }
-
-    public Builder eth1Endpoint(final Optional<String> eth1Endpoint) {
-      checkNotNull(eth1Endpoint);
-      this.eth1Endpoint = eth1Endpoint.filter(s -> !s.isBlank());
+    public Builder eth1Endpoints(final List<String> eth1Endpoints) {
+      checkNotNull(eth1Endpoints);
+      this.eth1Endpoints = eth1Endpoints;
+      if (!this.eth1Endpoints.isEmpty()) {
+        this.eth1Endpoints =
+            eth1Endpoints.stream().filter(s -> !s.isBlank()).collect(Collectors.toList());
+      }
```

Comment:
nit: Can probably just simplify this to always do the filtering - if the list is empty the filtering doesn't execute anyway.

```suggestion
      this.eth1Endpoints =
          eth1Endpoints.stream().filter(s -> !s.isBlank()).collect(Collectors.toList());
```